src/features/amUI/userRightsPage/UserRightsPage.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { useDocumentTitle } from '@/resources/hooks/useDocumentTitle';
+import { useTranslation } from 'react-i18next';
+
+import { Heading } from '@digdir/designsystemet-react';
+
+import { FakePageWrapper } from '../common/FakePageWrapper';
+import { useNavigate, useParams } from 'react-router-dom';
+import { UserIcon } from '@/components/UserIcon/UserIcon';
+import classes from './UserRightsPage.module.css';
+import { PageContainer } from '../common/PageContainer/PageContainer';
+import { PageWrapper } from '@/components';
+import { useGetUserByUUIDQuery } from '@/rtk/features/lookup/lookupApi';
+import { useGetReporteeQuery } from '@/rtk/features/userInfo/userInfoApi';
+import { amUIPath } from '@/routes/paths';
+
+export const UserRightsPage = () => {
+  const { t } = useTranslation();
+  const { id } = useParams();
+
+  const navigate = useNavigate();
+
+  const { data: reportee } = useGetReporteeQuery();
+  const { data: user } = useGetUserByUUIDQuery(id ?? '');
```

Comment:
Also, do you think you can please add a couple of the people from Diskret Nær Tiger AS to the parties.json in the Mock project in the BFF? Just to make testing easier here and in the future? 🙏

You can for example add Diane Davis, Narnia Economics, and Santa Claus (just make sure that the partyuuid and other data is the same). That way we have the three main cases covered: Someone with roles in the org that is not you (Diane Davis), an org with rights (Narnia Economics), and someone that inherits rights through an org (Santa Claus).